test/js-api/limits.js

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright 2018 WebAssembly Community Group participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+let kJSEmbeddingMaxTypes = 1000000;
+let kJSEmbeddingMaxFunctions = 1000000;
+let kJSEmbeddingMaxImports = 100000;
+let kJSEmbeddingMaxExports = 100000;
+let kJSEmbeddingMaxGlobals = 1000000;
+let kJSEmbeddingMaxDataSegments = 100000;
+
+let kJSEmbeddingMaxMemoryPages = 65536;
+let kJSEmbeddingMaxModuleSize = 1024 * 1024 * 1024;  // = 1 GiB
+let kJSEmbeddingMaxFunctionSize = 7654321;
+let kJSEmbeddingMaxFunctionLocals = 50000;
+let kJSEmbeddingMaxFunctionParams = 1000;
+let kJSEmbeddingMaxFunctionReturns = 1;
+let kJSEmbeddingMaxTableSize = 10000000;
+let kJSEmbeddingMaxTableEntries = 10000000;
```

Comment:
This should probably be called `kJSEmbeddingMaxElementSegments` based on the test below.

Reply:
Done